pkg/api/handlers/authprovider.go

```diff
@@ -143,6 +149,10 @@ func (ap *AuthProviderHandler) Configure(req api.Context) error {
 	}
 	envVars[providers.CookieSecretEnvVar] = cookieSecret
 
+	if ap.postgresDSN != "" {
+		envVars[providers.PostgresConnectionEnvVar] = ap.postgresDSN
+	}
```

Comment:
nit: Is there any harm in always including this?

Reply:
I don't think so? I guess I could remove the if statement. Doesn't really make a difference.